hail/www/template.xslt

```diff
@@ -94,17 +94,15 @@
                         (function () {
                             var cpage = location.pathname;
 
-                            if (cpage === "" || cpage === "/" || cpage === "/index.html") {
+                            if (cpage === "/" || cpage === "/index.html") {
                                 document.getElementById('hail-navbar-brand').className = "active";
                                 return;
                             };
 
                             var menuItems = document.querySelectorAll('#hail-menu a');
 
                             for (var i = 0; i < menuItems.length; i++) {
-                                var href = menuItems[i].pathname;
-
-                                if (href === cpage) {
+                                if (menuItems[i].pathname === cpage && menuItems[i].host == location.host) {
```

Comment:
location.host check needed to avoid matching equal pathnames for two different pages, i.e `google.com/docs` and `hail.is/docs`. This currently would only get triggered on `/` and `https://blog.hail.is/` and get caught in the home-page condition, but better to avoid.

Reply:
Yeah that thing only exists on powered-science, the docs, and the home page.

I think I agree with you if it is your intention for this javascript to only be used on pages served at `hail.is`?